### ISSUE TITLE
Card blanks whole view in HA 2022.12

### ISSUE THREAD

**Issue by peteh** [nielsfaber/scheduler-card#646]
### Checklist

- [X] I checked for similar existing issues (both open and closed) before posting.
- [X] I will participate in further discussion about this issue and can help by testing (if requested).

### Card Version

2.4.0

### Component Version

3.2.14

### Bug description

I think it's not the same issue as https://github.com/nielsfaber/scheduler-card/issues/642

I have a view that contains the scheduler card. The view is essentially empty when the card is contained. When I remove the card the view is showing normally again. 
Screenshot of the blank view: 
![image](https://user-images.githubusercontent.com/918728/208769330-24d5d63a-64cb-4491-b49d-de655d32df5f.png)

Not sure how I can debug more, the docker container logs are empty I think. 

HA: 2022.12
Browser: Firefox

Here is the raw code of the view: 
```
views:
  - theme: Backend-selected
    title: Home
    type: custom:masonry-layout
    badges: []
    cards:
      - square: false
        columns: 2
        type: grid
        cards:
          - type: thermostat
            entity: climate.living_room_heater
          - type: thermostat
            entity: climate.office_heater
          - type: thermostat
            entity: climate.bedroom_heater
          - type: thermostat
            entity: climate.toilet_heater
      - show_name: true
        show_icon: true
        type: button
        tap_action:
          action: toggle
        entity: input_boolean.automatic_heating
        show_state: true
      - type: entities
        entities:
          - entity: sensor.balcony_climate_temperature
          - entity: sensor.living_room_climate_temperature
          - entity: sensor.office_climate_temperature
          - entity: sensor.bedroom_climate_temperature
        show_header_toggle: false
      - type: custom:scheduler-card
        include:
          - climate.bedroom_heater
          - climate.living_room_heater
          - climate.office_heater
          - climate.toilet_heater
          - input_boolean.automatic_heating
        exclude: []
```

Any ideas what I could check? 
I tried: 
 * restarting HA
 * clearing browser cache
 * Try in Private mode of Firefox

### Steps to reproduce

Update to HA 2022.12
Update the card to 2.4.0
install the scheduler extension
Try to add the card to a view
Result: blank page

### Additional info
Maybe this console output from javascript helps: 
![image](https://user-images.githubusercontent.com/918728/208788306-baa3b4be-262e-4f6e-8cf2-fb87503e795d.png)

_No response_

**Comment by nielsfaber** [comment]
> Card Version
2.4.4

This is weird.
I released version 2.4.0 this week, there is no 2.4.4 yet.

> Maybe this console output from javascript helps:

It does! There is a severe error in the code which causes this.
I'm surprised it didn't show up in my tests, but I will have to make a patch for this asap.

I will let you know.

**Comment by peteh** [comment]
Thank you, let me know if you need more info or something reproduced. 

For the card version, it is shown to me like this in HACS: 

![image](https://user-images.githubusercontent.com/918728/208912406-f779a099-400d-4650-93d1-8ded8bfe4cdb.png)

**Comment by nielsfaber** [comment]
I did not manage to reproduce the problem.
However I made an attempt to fix this issue.
You would help me a lot if you could test it out locally.
Please overwrite the code of the card in the www/ folder by the attached file.
To verify the browser uses the modified code, please open the browser cache.
It should state a 'b' version:
![image](https://user-images.githubusercontent.com/16861605/209202238-0cd06c20-aad3-4bf3-8e92-e0af13ebe5db.png)
If this is not the case, please empty the cache and retry (or click 'reload resources' in the HA dashboard).

[scheduler-card.js.zip](https://github.com/nielsfaber/scheduler-card/files/10289500/scheduler-card.js.zip)

**Comment by bertreb** [comment]
I also had the blank view with version 2.4.0. I replaced scheduler-card.js with version 2.4.0b
The problem seems to be solved in my case. The view is visible and the new popup settings window apears.

**Comment by nielsfaber** [comment]
Thanks for testing. I now released v2.4.1 with this change.
Hope it works OK for everyone.

**Comment by peteh** [comment]
Hi, 

was a bit late to the game. I can confirm 2.4.1 works (updated through HACS): 
![image](https://user-images.githubusercontent.com/918728/209374023-14b8f55e-2408-44b9-a490-c608abf60203.png)

I'll close the issue. Thank you very much for the quick fix.
